src/CommonITILValidation.php

```diff
@@ -670,7 +670,20 @@ public static function getNumberToValidate($users_id)
             'COUNT'  => 'cpt',
             'WHERE'  => [
                 'status'             => self::WAITING,
-                'users_id_validate'  => $users_id
+                'users_id_validate'  => $users_id,
+                'NOT' => [
+                    'AND' => [
+                        [
+                            'tickets_id' => new QuerySubQuery([
+                                'SELECT' => 'id',
+                                'FROM'   => 'glpi_tickets',
+                                'WHERE'  => [
+                                    'status' => CommonITILObject::CLOSED
+                                ]
+                            ])
+                        ]
+                    ]
+                ]
```

Comment:
I think the query is wrong (before your PR), because it counts the number of pending validations, when it should count the number of tickets with (one or more) pending validations.
Deleted tickets should also be excluded.